.github/workflows/docs.yml

```diff
@@ -40,6 +40,8 @@ jobs:
       - uses: dtolnay/rust-toolchain@master
         with:
           toolchain: ${{ env.NIGHTLY_TOOLCHAIN }}
+      
+      - uses: dtolnay/install@cargo-docs-rs
```

Comment:
don't use this action to install it, use the same as the rest of CI

Reply:
I've switched it to use `cargo install` instead in [b3217e0](https://github.com/bevyengine/bevy/pull/14350/commits/b3217e09f249e19ea528bac6fe83b8d591758889)